GoogleForCreators/web-stories-wp
# Removed button titles where irrelevant (which is everywhere)

## Context

Removed button titles from floating menu buttons (both text and icon buttons) and one other instance (lock toggle in style panel).

I watched `document.querySelectorAll("button[title]")` while navigating around the app, and didn't find anything else, so I think we're good.

## User-facing changes

| Before | After |
|-|-|
| <img width="640" alt="Screen Shot 2022-12-21 at 09 56 38" src="https://user-images.githubusercontent.com/637548/208863271-d70e61de-2878-4b66-9493-0ed09b0bee7b.png"> | <img width="640" alt="Screen Shot 2022-12-21 at 09 48 48" src="https://user-images.githubusercontent.com/637548/208863371-6ab3b508-8e81-48ef-a937-99f991cbb91f.png"> |
| <img width="640" alt="Screen Shot 2022-12-21 at 09 48 16" src="https://user-images.githubusercontent.com/637548/208863443-634d882b-67b1-4371-8100-90f45b1965bf.png"> | <img width="640" alt="Screen Shot 2022-12-21 at 09 48 35" src="https://user-images.githubusercontent.com/637548/208863487-bf6f3f8e-f08e-4217-abf8-2dcddc9277e0.png"> |

## Testing Instructions

This PR can be tested by following these steps:

1. Long-hover any button in the editor (most importantly in the design menu)
2. Observe that the browser-native tooltip does not show up

## Checklist




---

Fixes #12252
